app/src/main/res/drawable/edit_text_background.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.edit_text_background` appears to be unused
 Unused resources make applications larger and slow down builds.